.eslintrc.js

```diff
@@ -0,0 +1,246 @@
+module.exports = {
```

Comment:
I copied `.eslintrc`, added `module.exports`, and fixed some of the comments in order to comply with linting (some of them didn't have a space between `//` and the comment string)